### ISSUE TITLE
Can I add GPU Mining from Panel?

### ISSUE THREAD

**Issue by dushLK** [UnamSanctam/SilentCryptoMiner#1969]
Hi, I've successfully using the panel to monitor and change the miners which I created for monero mining. Can I add gpu mining settings to them via the panel? If so, How can I add 2 json settings on the panel configuration?

Thanks.

**Comment by UnamSanctam** [comment]
No, if you built the miner without a GPU miner added then the GPU miner code isn't present inside your miner program.
